src/components/ha-selector/ha-selector-select.ts

```diff
@@ -25,15 +27,17 @@ export class HaSelectSelector extends LitElement {
       naturalMenuWidth
       .label=${this.label}
       .value=${this.value}
+      .helper=${this.helper}
       .disabled=${this.disabled}
       @closed=${stopPropagation}
       @selected=${this._valueChanged}
     >
-      ${this.selector.select.options.map(
-        (item: string) => html`
-          <mwc-list-item .value=${item}>${item}</mwc-list-item>
-        `
-      )}
+      ${this.selector.select.options.map((item: string | SelectOption) => {
+        const value = typeof item === "object" ? item.value : item;
+        const label = typeof item === "object" ? item.label : item;
```

Comment:
Or should we do it like `ha-form`? 🤔 

<https://github.com/home-assistant/frontend/blob/dev/src/components/ha-form/ha-form-multi_select.ts#L24-L30>

Reply:
I like the object better, but not sure how well it translates to the backend